packages/mem_usage/mem_usage.0.1.1/opam

```diff
@@ -0,0 +1,36 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Cross-platform stats about memory usage"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Romain Beauxis <toots@rastageeks.org>"]
+license: "GPL-2.0"
```

Comment:
I guess this should be `GPL-2.0-only` or `GPL-2.0-or-after`

Reply:
It's actually MIT. Fixed upstream, thanks for spotting!